test/unit/group-controller.test.js

```diff
@@ -0,0 +1,335 @@
+import 'babel-polyfill';
+import { v4 as uuidv4 } from 'uuid';
+import GroupService from '../../src/service/group-service';
+import {
+  addGroup, addUsersToGroup, getGroups, getGroupById, updateGroup, deleteGroup,
+} from '../../src/controllers/group-controller';
+
+jest.mock('../../src/service/group-service', () => ({
+  getAll: jest.fn(),
+  getById: jest.fn(),
+  addUsersToGroup: jest.fn(),
+  add: jest.fn(),
+  update: jest.fn(),
+  delete: jest.fn(),
+}));
+
+
```

Comment:
Please apply this best practices for tests:
1) group tests by `describe` blocks - it will increase readability 
2) move common parts to beforeAll, afterAll, beforeEach, afterEach methods (like response mock initialization in your case) - it will remove code duplications
3) clear mocks in afterAll or afterEach - so it will reset mock to initial state each time, otherwise it may lead to incorrect behavior in case of mocks are mutated in tests (in current test it is not actual, but it is better to always remember about this);

```
describe('group controller', ()=>{
   let response

   beforeEach(()=>{
      response = {
        status: jest.fn(() => response),
        json: jest.fn(),
        send: jest.fn(),
   };})

   afterEach(()=>{
      jest.clearAllMocks();
   }))
   describe('get all groups method', ()=>{
      test('should return all groups', ()=>{ ... })
      test('should return empty array if there are no groups', ()=>{ ... }) 
   })
})
```